mssql_query_block.py

```diff
@@ -1,53 +1,49 @@
-from nio.properties import VersionProperty
+from nio.properties import VersionProperty, StringProperty
 from nio.block.terminals import output
-from .mssql_base import MSSQLBase
-from .mssql_conditions import MSSQLConditions
 from nio.block.mixins.enrich.enrich_signals import EnrichSignals
 
+from .mssql_base import MSSQLTabledBase
+from .mssql_conditions import MSSQLConditions
+
 
 @output('results', label='Results')
 @output('no_results', label='No Results')
-class MSSQLQuery(EnrichSignals, MSSQLBase, MSSQLConditions):
-
-    version = VersionProperty("0.2.0")
-
-    def process_signals(self, signals):
-        if self.isConnecting:
-            self.logger.error(
-                'Connection already in progress. Dropping signals.')
+class MSSQLQuery(EnrichSignals, MSSQLTabledBase, MSSQLConditions):
+
+    version = VersionProperty("1.0.1")
+
+    def process_signals(self, signals, **kwargs):
```

Comment:
Just curious, why did you add `**kwargs` here?

Reply:
And it looks like in other process_signals calls too

Reply:
This probably me and probably a PyCharm Pro auto-fix/auto-formatting to match the signature type of the overriding member; the parent signature is ```def process_signals(self, signals, input_id=DEFAULT_TERMINAL):```

So, probably overkill and I missed it on my clean-up.

Reply:
I removed them... just self, signals now